### PR TITLE
NameError name 'Fraction' is not defined

### PR DESCRIPTION
**Pull request by sv99**

### CODE REVIEW

File: picamera/camera.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def to_rational(value):
                         "int, float, Decimal, or Fraction instead"))
             except (TypeError, ValueError):
                 # try and convert anything else (e.g. Decimal) to a Fraction
-                value = Fraction(value)
```
Comment (sv99) [inline picamera/camera.py]: may be misprint?

**Comment by waveform80** [comment]
Doh! You're absolutely right. Many thanks for the PR, and for including a test-case for it. In this particular case it's probably easier to simply add the test to test_framerate in tests/test_attr.py rather than create a whole new module for the unit-test (in picamera I tend to go for functional tests rather than unit tests, simply because they tend to expose changes in the underlying firmware rather nicely). Is there any chance I could be horribly demanding and ask you to alter the PR accordingly?

**Comment by sv99** [comment]
PR changed.

**Comment by waveform80** [comment]
Excellent, many thanks!

**Pull request merged**